config/runtime.exs

```diff
@@ -116,7 +116,7 @@ if config_env() == :prod || config_env() == :dev do
     plugins: [
       {Oban.Plugins.Cron,
        crontab: [
-         {"@midnight", Mindwendel.Worker.RemoveBrainstormingsAfterPeriodWorker}
+         {"@midnight", Mindwendel.Worker.RemoveBrainstormingsAndUsersAfterPeriodWorker}
```

Comment:
We should have two seperate jobs that we can execute / trigger individually.

Reply:
I thought about it before implementing this, and i opted against it. Reason: This is all one cleanup job with one purpose and one use-case. I dont see any benefit splitting those up. They should be running at about the same time. I dont think its good practice to just create thousands of mini jobs, just because you can.